<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvertInfoDialog</class>
 <widget class="QDialog" name="AdvertInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#dialogbg{
    border:1px solid #none;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #edeeef, stop: 1.0 #d4d7da);
}
QScrollArea{
	border:none;
	background:#00000000;
}
QWidget#center,#stackedWidget,#contentPage,#emptyPage
{
	border:none;
	background:#00000000;
}


QWidget{
    font-family:&quot;微软雅黑&quot;;
	color:#666666;
}

QWidget#spliteup,#splitedown
{
	border:1px solid  #D3D3D3;
}
QPushButton#pbClose{
	border:none;
	height:27px;
	width:30px;
	max-width:30px;
	min-width:30px;
	 
	max-height:27px;
	min-height:27px;

	background:#00000000;
	border-image: url(:/Images/close_normal.png); 
}
QPushButton#pbClose:hover{
	background:#F25561;
	border-image: url(:/Images/close_hover.png);  
}
QPushButton#pbClose:pressed{
	background:#E74450;
	border-image: url(:/Images/close_hover.png);  
}

QPushButton#pbSearch,#pbSend{
	border:none;
	height:23px;
	width:35px;
	color:#FFFFFF;
    background-color:#3DCE3D;
}
QPushButton#pbSearch:hover,#pbSend:hover{
	background-color:#38C738;
}
QPushButtonpbSearch:pressed,#pbSend:pressed{
	background-color:#2FB82F;
}

QPushButton#pbCancel{
    color:#666666;
    border:1px solid #E5E5E5;
}
QPushButton#pbCancel:hover{
    background-color:#f5f5f5;
}
QPushButton#pbCancel:pressed{
    background-color:#EBEBEB;
}

QLabel#producttag,#producttype
{
	border:none;
	color:#FFFFFF;;
	font-weight:bold;
	font-size:20px;
	
}

QWidget#tagbk{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1
                     stop: 0 #00000000,
					 stop: 1.0 #D0333333); 
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dialogbg" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="titlebar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="center">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>380</width>
           <height>602</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>请输入产品ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_productId">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSearch">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QWidget" name="contentPage">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_productInfo" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>111</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>111</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="label_productInfo">
                   <property name="text">
                    <string>请输入产品信息介绍:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="textEdit"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="spliteup" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>1</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>1</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="picbk" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>252</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>252</height>
                 </size>
                </property>
                <widget class="QLabel" name="productpic">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>16</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
                <widget class="QWidget" name="tagbk" native="true">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>201</y>
                   <width>20</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>20</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>20</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="producttype">
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>-1</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="producttag">
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>-1</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_title">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>20</number>
                 </property>
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <property name="maximumSize">
                    <size>
                     <width>70</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_markprice_title">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>28</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_qunarprice_title">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>28</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>15</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_markprice">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_qunarprice">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="emptyPage">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>89</number>
              </property>
              <property name="topMargin">
               <number>50</number>
              </property>
              <property name="rightMargin">
               <number>89</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_emptyimg">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="pixmap">
                 <pixmap resource="localman.qrc">:/Images/no_session_selected_background.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_emptytext">
                <property name="text">
                 <string>搜索一个产品吧~</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="splitedown" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottompannel" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>175</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbCancel">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSend">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
